deploy/playbooks/roles/common/tasks/nginx.yml

```diff
@@ -33,6 +33,27 @@
   action: file src=/etc/nginx/sites-available/{{ app_name }}.conf dest=/etc/nginx/sites-enabled/{{ app_name }}.conf state=link
   sudo: true
 
+- name: ensure ssl certs directory
+  action: file dest=/etc/ssl/certs state=directory
+  sudo: true
+
+- name: ensure ssl private directory
+  action: file dest=/etc/ssl/private state=directory
+  sudo: true
+
+- name: create self-signed SSL cert
+  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Oregon/L=Portland/O=IT/CN=${app_name}" -days 3650 -keyout /etc/ssl/private/{{ ansible_fqdn }}.key -out /etc/ssl/certs/{{ ansible_fqdn }}-bundled.crt -extensions v3_ca creates=/etc/nginx/ssl/{{ ansible_fqdn }}-bundled.crt
+  when: development_server
```

Comment:
Is this going to regenerate a new SSL cert and key file every time Ansible runs?